### ISSUE TITLE
AutoStandby only works on half of the time

### ISSUE THREAD

**Issue by zwim** [koreader/koreader#9238]
* KOReader version: 2022.05.01
* Device:Sage

#### Issue
Poowersavings. Time in autosuspend is way to low.

#### Steps to reproduce
restart koreader , wait for a minute, check systemstat and see that the device is only about half of the time in standby. Iit should be there for at least 80% - 90% of the time.

A crash log will follow, when I am at home again.

**Comment by NiLuJe** [comment]
FWIW, the only failures I've seen on my end are explicitly because the touchscreen is busy (much more common no the Elipsa with the pen stuck in the cover).

**Comment by NiLuJe** [comment]
Yeah, I'm (vertically) semi-stacking the Sage on top of the Elipsa on my nightstand, and I'm not quite sure how I've managed to avoid weirdness like that so far :D.
